src/test/java/seedu/taassist/commons/util/AppUtilTest.java

```diff
@@ -1,17 +1,11 @@
 package seedu.taassist.commons.util;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static seedu.taassist.testutil.Assert.assertThrows;
 
 import org.junit.jupiter.api.Test;
 
 public class AppUtilTest {
 
-    @Test
-    public void getImage_exitingImage() {
-        assertNotNull(AppUtil.getImage("/images/ta_assist_32.png"));
-    }
-
```

Comment:
Since the actual function is not removed, we can perhaps change the test case to use another image instead of removing the test case entirely.